styles/componentStyles/MoreInfoModal.module.css

```diff
@@ -0,0 +1,34 @@
+.darkBG {
+  background-color: grey;
+  opacity: 0.5;
+  position: fixed;
+  width: 150vw !important;
+  height: 150vh;
+  margin: none;
+}
```

Comment:
We can get rid of this I am pretty sure. I THINK this is the one that I replaced with the global calls